package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "cytoscape-klayjs",
+  "version": "0.0.0",
+  "description": "KLayJS for Cytoscape",
+  "main": "cytoscape-klayjs.js",
+  "spm": {
+    "main": "cytoscape-klayjs.js"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cytoscape.js-KLayJS.git"
+  },
+  "keywords": [
+    "cytoscape",
+    "cyext"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/cytoscape.js-KLayJS/issues"
+  },
+  "homepage": "https://github.com/cytoscape.js-KLayJS",
```

Comment:
klay should be a "dependency" https://docs.npmjs.com/files/package.json#dependencies